src/FirmwarePlugin/APM/APMFirmwarePlugin.cc

```diff
@@ -890,9 +890,18 @@ void APMFirmwarePlugin::startMission(Vehicle* vehicle)
 
     if (!vehicle->armed()) {
         // First switch to flight mode we can arm from
-        if (!_setFlightModeAndValidate(vehicle, "Guided")) {
-            qgcApp()->showAppMessage(tr("Unable to start mission: Vehicle failed to change to Guided mode."));
-            return;
+        // In Ardupilot for vtols and airplanes we need to set the mode to auto and then arm, otherwise if arming in guided
+        // If the vehicle has tilt rotors, it will arm them in forward flight position, being dangerous.
+        if (vehicle->fixedWing()) {
+            if (!_setFlightModeAndValidate(vehicle, "Auto")) {
+                qgcApp()->showAppMessage(tr("Unable to start mission: Vehicle failed to change to Auto mode."));
```

Comment:
Do we need a change to the translation files  for this?

Reply:
Good point. it is reusing a message already present, but we probably need to update translation files. Let me a couple days to fix it. Thanks for spotting it!